client/src/common/components/ListViewItem.jsx

```diff
@@ -71,7 +80,15 @@ class ListViewItem extends PureComponent {
             })}
             disabled={pending}
             onClick={this.handleFavClick}
-            title={`${isFavourite ? 'Remove from' : 'Add to'} favourites`}
+            title={
```

Comment:
change to:
`formatMessage({ id: isFavourite ? 'common.list-view-item.remove-fav' : 'common.list-view-item.add-fav' })`